src/dlgProfilePreferences.cpp

```diff
@@ -1003,10 +1003,10 @@ void dlgProfilePreferences::initWithHost(Host* pHost)
         }
     }
 
-    comboBox_controlCharacterHandling->setItemData(0, ControlCharacterMode::AsIs);
-    comboBox_controlCharacterHandling->setItemData(1, ControlCharacterMode::Picture);
-    comboBox_controlCharacterHandling->setItemData(2, ControlCharacterMode::OEM);
-    auto cch_index = comboBox_controlCharacterHandling->findData(pHost->getControlCharacterMode());
+    comboBox_controlCharacterHandling->setItemData(0, QVariant::fromValue(ControlCharacterMode::AsIs));
```

Comment:
Are the `QVarient::fromValue(...)` needed here? Looking at the Qt Documentation it says:
> **`[static] QVariant QVariant::fromValue(const T &value)`**
>
> Returns a `QVariant` containing a copy of value. Behaves exactly like `setValue()` otherwise.
> Example:
```cpp
  MyCustomStruct s;
  return QVariant::fromValue(s);
```
>
> Note: If you are working with custom types, you should use the `Q_DECLARE_METATYPE()` macro to register your custom type.
>
> See also `setValue()` and `value()`

But given that `QComboBox::setItemData(...)` takes a `const` reference to a ``QVariant`, perhaps it is needed here after all...

:man_shrugging:

Reply:
Yeah, it seemed to be necessary.